enterprise/api/v1/urls.py

```diff
@@ -7,16 +7,15 @@
 
 from enterprise.api.v1 import views
 
-# Pylint considers module level variable as "Constant" and expects them to be upper-cased
-# that is why we have disabled 'invalid-name' check for variable definition below.
 router = DefaultRouter()  # pylint: disable=invalid-name
 router.register("site", views.SiteViewSet, 'site')
 router.register("auth-user", views.UserViewSet, 'auth-user')
-router.register(
-    "enterprise-course-enrollment",
-    views.EnterpriseCourseEnrollmentViewSet,
-    'enterprise-course-enrollment'
-)
+router.register("enterprise-customer-catalog", views.EnterpriseCustomerCatalogViewSet, 'enterprise-customer-catalog')
+router.register("enterprise-catalogs", views.EnterpriseCustomerCatalogApiViewSet, 'enterprise-catalogs')
```

Comment:
Right, apologies if this was not clear in our previous discussions. `/enterprise/api/v1/enterprise-catalogs` should return the serialized list of `EnterpriseCustomerCatalog` models that the requesting user has access to. We have no need for an endpoint that returns the results of an unfiltered query to the catalog's search endpoint.

How should we proceed? Is this something you can address as a part of the program landing page work or should I file a separate ticket?

FYI @brittneyexline